### DISCUSSION TITLE
2023-01-22 - January 22, 2023

### DISCUSSION THREAD

**Discussion by dprincz** [MESH-Model/MESH-Dev#28] [Developer Meetings]
Discussion

- Confirmed path forward, documented in #27, for attendees which weren't able to attend
- Discussed HPC strategy in coordination with UofC
- Confirmed that no vector-based version exists for MRB, SRB/Nelson-Churchill

Actions

- Still to confirm with Zelalem there aren't other local changes relative to the r1860_ME code
- Still to confirm @AlaBahrami is able to complete Fraser runs using the r1860_ME code

**Comment by AlaBahrami** [comment]
Thanks @dprincz for mentioning. I'll launch series of runs based on the `r1860_ME` version of code for the application of **Fraser** production runs. I'll keep you posted on that front.
